pulsar-client-cpp/lib/Version.h

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2016 Yahoo Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef LIB_VERSION_H_
+#define LIB_VERSION_H_
+
+#ifndef _PULSAR_VERSION_
+#define _PULSAR_VERSION_ "1.17"
```

Comment:
Oh, I see. Then maybe grepping for the version could be done directly inside the `CMakelist.txt` so that it works even when not using build.sh script (not sure exactly how that works though).